### PR TITLE
Package zipperposition.1.5

### PR DESCRIPTION
**Pull request by c-cube**
### `zipperposition.1.5`

A fully automatic theorem prover for typed first-order and beyond.

Zipperposition is intended to be a superposition prover for full first
order logic, plus some extensions (datatypes, recursive functions, arithmetic
for integers and rationals, higher-order, induction).
The accent is on flexibility, modularity and simplicity rather than
performance, to allow quick experimenting on automated theorem proving. It
generates TSTP traces or graphviz files for nice graphical display.

---
* Homepage: https://github.com/c-cube/zipperposition
* Source repo: https://github.com/c-cube/zipperposition.git
* Bug tracker: https://github.com/c-cube/zipperposition/issues

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

File: packages/zipperposition/zipperposition.1.5/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/zipperposition"
+bug-reports: "https://github.com/c-cube/zipperposition/issues"
+tags: ["logic" "unification" "term" "superposition" "prover"]
+dev-repo: "https://github.com/c-cube/zipperposition.git"
+build: ["jbuilder" "build" "@install"]
+build-doc: ["jbuilder" "build" "@doc"]
+depends: [
+  "ocamlfind" {build}
```
Comment (jpdeplaix) [inline packages/zipperposition/zipperposition.1.5/opam]: I think this dependency is not needed.

**Comment by c-cube** [comment]
Seriously, the macOS build takes literally days…

**Comment by hcarty** [comment]
Merging since everything other than the Travis macOS is working. @c-cube maybe the extraneous findlib dependencies can be removed from these recent packages in a follow-up PR?

**Comment by c-cube** [comment]
Next release, yeah. A dep to findlib isn't much, really -_-

**Pull request merged**